packages/devtools-network-console/src/ui/ResponseViewer/index.tsx

```diff
@@ -154,7 +155,7 @@ export function ResponseViewer(props: IResponseViewerProps) {
         setCurrentTab('body');
     }
 
-    if (props.response.isWebsocketUpgrade) {
+    if (props.showWSView) {
```

Comment:
We should consider selecting the WebSocket view by default the first time. This is accomplished by using a ref + effect. It's something like this....

```jsx
const showWsRef = useRef<boolean>(null);
useEffect(() => {
    if (!showWsRef.current && props.showWSView) {
        setCurrentTab('websocket');
        showWsRef.current = true;
    }
}, [showWsRef, props.showWSView, setCurrentTab]);
```

Reply:
Cool! I'll see if I can get that working. Now that we listen to the ws connected host message, the tab might be slightly delayed, so I'll make sure this isn't too jarring of a switch

Reply:
Discussed offline. Adding as a todo for now and will be a follow up change